static/css/common.css

```diff
@@ -1,3 +1,7 @@
-.hidden {
+div {
+  display: flex;
```

Comment:
That's going to surprise people that don't know about our divs, right?, or did I misunderstand a scope? `div.flex { ... }`?